common/lib/capa/capa/tests/response_xml_factory.py

```diff
@@ -699,6 +699,8 @@ def create_response_element(self, **kwargs):
 
             *additional_answers*: list of additional asnwers.
 
+            *old_answers*: list of additional answers to be coded in old format
```

Comment:
Sure, I'll call it the pre_attribute.. and put a comment explaining what's going on.